app/adapters/organization.js

```diff
@@ -1,15 +1,10 @@
-import DRFAdapter from './drf';
-import ENV from 'irene/config/environment';
-import IreneAdapterMixin from 'irene/mixins/data-adapter-mixin';
+import commondrf from './commondrf';
 
-export default DRFAdapter.extend(IreneAdapterMixin, {
-  host: ENV.host,
-  namespace: ENV.namespace,
-  addTrailingSlashes: false,
+export default class Organization extends commondrf {
   async saveReportPreference(modelInstance, data) {
     const modelId = modelInstance.get('id');
     const url = this.buildURL('organization', modelId) + '/report_preference';
```

Comment:
@cosmosgenius Can you confirm whether `this.buildURL` provide the correct host?

Reply:
Tested....it is working!